03/Category.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react';
 
 function Category(props) {
-    return <h2>Category</h2>
+  const list = React.Children.map(props.children, (child) => {
```

Comment:
Trochę nadużywasz tego rozwiązania.
Prościej i wygodniej jest po prostu przekazać element z `<li>`. 
A wew. tego komponentu po prostu wykorzystać {props.children}.
Oczywiście to co jest zrobione jest ok ;)